llvm/tools/llvm-exegesis/lib/RISCV/Target.cpp

```diff
@@ -864,10 +864,17 @@ Error ExegesisRISCVTarget::randomizeTargetMCOperand(
     // 5-bit unsigned immediate value.
     AssignedValue = MCOperand::createImm(randomIndex(31));
     break;
+  case RISCVOp::OPERAND_SIMM12_LO:
+  case RISCVOp::OPERAND_UIMM20_LUI:
+  case RISCVOp::OPERAND_UIMM20_AUIPC:
+  case RISCVOp::OPERAND_BARE_SIMM32:
+    AssignedValue = MCOperand::createImm(0);
```

Comment:
I guess you tried to make these switch `case`s explicit? otherwise this should have the same effect as falling into the `default` case

Reply:
oh nvm, you split it out of immediate